lib/publiccloud/provider.pm

```diff
@@ -384,22 +385,25 @@ sub terraform_apply {
     record_info('TFM cmd', $cmd);
 
     script_retry($cmd, timeout => $terraform_timeout, delay => 3, retry => 6);
-    my $ret = script_run('terraform apply -no-color -input=false myplan', $terraform_timeout);
-    unless (defined $ret) {
-        if (is_serial_terminal()) {
-            type_string(qq(\c\\));    # Send QUIT signal
+    my $tries = is_azure() ? 3 : 1;    # Retry only on azure platform
```

Comment:
ah I forgot that we agreed on to limit this to QAM jobs only so actually it should be 
```suggestion
    my $tries = is_azure() && get_var('PUBLIC_CLOUD_QAM') ? 3 : 1;    # Retry only on azure platform and only for QAM jobs 
```

Reply:
Done.